.travis.yml

```diff
@@ -12,14 +12,15 @@ rvm:
   - 2.3.8
   - 2.4.5
   - 2.5.3
+  - 2.6.0
```

Comment:
Bundler is still failing on 2.6, working on that over in https://github.com/bundler/bundler/pull/6856